src/translators/timescale.py

```diff
@@ -114,6 +114,10 @@ def dispose(self):
 
     def sql_error_handler(self, exception):
         analyzer = PostgresErrorAnalyzer(exception)
+        err_msg = analyzer.is_aggregation_error()
+        if err_msg:
+            logging.error(err_msg)
```

Comment:
maybe we don't need this logging, we have already a log in the place were we invoke the function

Reply:
removed